omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -54,7 +54,7 @@
         var widgetConfig = {
             initialPatients: lastViewedPatients,
             doInitialSearch: ${ doInitialSearch ? "\"" + ui.escapeJs(doInitialSearch) + "\"" : "null" },
-            minSearchCharacters: ${ config.minSearchCharacters ?: 3 },
```

Comment:
Did you accidentally change this from config.minSearchCharacters?

Reply:
I assumed that this was a bug that @ssmusoke discovered and was fixing in this commit... ie it wasn't picking up the parameter because it should have just been minSearchCharacters... however, admittedly, I didn't double-check this... :)

Reply:
@dkayiwa Nope that is the fix which uses the value from the fragment controller, @mogoodrich you are right that is the bug fix - removing the config parameter

Reply:
@dkayiwa Thank you for the additional search, only one location, in the patientSearchAndSelectWidget is applicable since its the only GSP. The rest as JS files which use the config values in the gsp